target/scripts/helpers/error.sh

```diff
@@ -62,13 +62,11 @@ function dms_panic__no_file { dms_panic 'no-file' "${1}" "${2}"; }
 function dms_panic__misconfigured { dms_panic 'misconfigured' "${1}" "${2}"; }
 function dms_panic__invalid_value { dms_panic 'invalid-value' "${1}" "${2}"; }
 
-# Call this method when you want to panic (emit a 'FATAL' log level error, and exit uncleanly).
+# Call this method when you want to panic (emit an 'ERROR' log and exit uncleanly).
 # `dms_panic` methods should be preferred if your failure type is supported.
 function _shutdown
 {
-  local FATAL_ERROR_MESSAGE=$1
-
-  _notify 'fatal' "${FATAL_ERROR_MESSAGE}"
-  _notify 'err' "Shutting down.."
+  _notify 'error' "${1}"
+  _notify 'error' 'Shutting down...'
   kill 1
 }
```

Comment:
In the earlier PR before extracting to this one, you originally had added an `exit 1` alongside the `kill 1` (_that was a little more explicit in signal used_), do you no longer see `exit 1` as beneficial here? (_I assumed `kill` was enough, but perhaps `exit` is still useful in this scenario?_)

Reply:
Done in [b022e20](https://github.com/docker-mailserver/docker-mailserver/pull/2486/commits/b022e20c4623bc12fbff24eb224715b560124c8e)